automated install/basic-install.sh

```diff
@@ -961,7 +967,6 @@ updatePihole() {
   CreateLogFile
   installPiholeWeb
   installCron
-  configureFirewall
```

Comment:
It's called on install, but not on update, since it's a whiptail dialog it would break --unattended if it fired during update.